### PR TITLE
some layer list item testing

### PR DESCRIPTION
**Pull request by CarlyLanglois**

### CODE REVIEW

File: __tests__/components/LayerListItem.test.js [hunk 1]
```diff
@@ -88,4 +91,107 @@ describe('LayerListItem', function() {
...
+  it('renders dragging item', function() {
+    var container = document.createElement('div');
+    var item = ReactDOM.render((
+      <LayerListItem index={0} moveLayer={function() {}} intl={intl} title={overlay.get('title')} map={map} layer={overlay} />
+    ), container);
+    var newItem = 'foo';
+    var actual = item.renderItem(newItem, true).type;
+    var expected = 'div';
+    assert.equal(actual, expected);
+    actual = item. renderItem(newItem, false);
```
Comment (wnordmann) [inline __tests__/components/LayerListItem.test.js]: Is there a space between `item.` and `renderItem` ?

File: __tests__/components/LayerListItem.test.js [hunk 2]
```diff
@@ -88,4 +91,107 @@ describe('LayerListItem', function() {
     ReactDOM.unmountComponentAtNode(container);
   });
 
+  it('renders dragging item', function() {
```
Comment (wnordmann) [inline __tests__/components/LayerListItem.test.js]: If this is testing item draging

File: __tests__/components/LayerListItem.test.js [hunk 3]
```diff
@@ -88,4 +91,107 @@ describe('LayerListItem', function() {
...
+    const actual = renderer.getRenderOutput().props.children[0].props.className;
+    const expected = 'layer-list-item';
+    assert.include(actual, expected);
+  });
+
+  it('renders the filter, table, style, and label modals', function() {
+    const renderer = TestUtils.createRenderer();
+    renderer.render(<LayerListItem index={0} moveLayer={function() {}} intl={intl} title={overlay.get('title')} map={map} layer={overlay}/>);
+    var filterModal = renderer.getRenderOutput().props.children[1].props.children[1].props.children[0];
+    assert.notEqual(filterModal, undefined);
```
Comment (wnordmann) [inline __tests__/components/LayerListItem.test.js]: would a better test be something like `assert.notEqual(filterModal, <filterModal/>);`
Might want to define <filterModal> better than example

File: __tests__/components/LayerListItem.test.js [hunk 4]
```diff
@@ -88,4 +91,107 @@ describe('LayerListItem', function() {
...
+    assert.equal(actual, expected);
+    actual = item.calculateInRange();
+    expected = true;
+    assert.equal(actual, expected);
+    ReactDOM.unmountComponentAtNode(container);
+    var item2 = ReactDOM.render((
+      <LayerListItem index={0} moveLayer={function() {}} intl={intl} title={layer.get('title')} map={map} layer={layer} handleResolutionChange={true}/>
+    ), container);
+    actual = item2.props.handleResolutionChange;
+    expected = true;
```
Comment (wnordmann) [inline __tests__/components/LayerListItem.test.js]: missing asset for this test

File: __tests__/components/LayerListItem.test.js (hunk 2 above)
Comment (CarlyLanglois) [inline __tests__/components/LayerListItem.test.js]: yeah, it checks to see if the div with the item is created based on the status of isDragging. if isDragging is false, no dragging item should be rendered

**Pull request merged**